src/v0/destinations/sendgrid/config.js

```diff
@@ -38,8 +38,9 @@ const TRACK_EXCLUSION_FIELDS = [
   "trackingSettings"
 ];
 
-const DELETE_CONTACTS_ENDPOINT = `${BASE_ENDPOINT}/marketing/contacts`;
+const DELETE_CONTACTS_ENDPOINT = `${BASE_ENDPOINT}/marketing/contacts?ids=[IDS]`;
 const MAX_BATCH_SIZE = 30000;
+const urlLimit = 8150;
```

Comment:
Not that there is a limit of max URL length in HTTP requests in general.

Reply:
Yeah 2048. But through the trial and error method I found it to be this.